src/signals/incident/definitions/wizard-step-1-beschrijf.js

```diff
@@ -61,7 +61,6 @@ export default {
           path: 'text',
           placeholder: 'Beschrijf uw melding',
           maxLength: 1000,
-          doNotUpdateValue: true,
```

Comment:
What was the purpose of the `doNotUpdateValue` prop and why wasn't it necessary anymore?